app/pb_kits/playbook/pb_currency/_currency.jsx

```diff
@@ -1,72 +1,75 @@
 /* @flow */
 
-import React from 'react'
-import classnames from 'classnames'
+import React from "react";
+import classnames from "classnames";
 
-import {
-  Body,
-  Caption,
-  Title,
-} from '../'
+import { Body, Caption, Title } from "../";
 
 type CurrencyProps = {
-  align?: 'left' | 'center' | 'right',
+  align?: "left" | "center" | "right",
   className: String,
-  currencySymbol: '$',
+  currencySymbol: "$",
```

Comment:
I believe this should be updated to type String.
And should it be changed to optional? (`currentSymbol?: String`)
You are setting the default value to "$" below.